ts/src/base/Exchange.ts

```diff
@@ -2283,23 +2284,11 @@ export default class Exchange {
     }
 
     async fetchOHLCV (symbol: string, timeframe = '1m', since: Int = undefined, limit: Int = undefined, params = {}): Promise<OHLCV[]> {
-        if (!this.has['fetchTrades']) {
-            throw new NotSupported (this.id + ' fetchOHLCV() is not supported yet');
+        let message = '';
+        if (this.has['fetchTrades']) {
+            message = '. If you want to build OHLCV candles from trade executions data, see an example at:  https://github.com/ccxt/ccxt/tree/master/examples/' + this.currentLanguageExtension + '/build-ohlcv-bars.' + this.currentLanguageExtension;
```

Comment:
can we change the message to something like: "check the example build-ohlcv-bars  inside https://github.com/ccxt/ccxt/tree/master/examples/` to avoid adding this `currentLanguageExtension` prop?

Reply:
oky sure

Reply:
done